packages/database/src/migrations/20210623220403-LESMISGrossIntakeRate-modifies-data.js

```diff
@@ -0,0 +1,429 @@
+'use strict';
+
+import { insertObject, deleteObject, generateId, findSingleRecord } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+/*
+ * DATA
+ */
+
+// Generate an inclusive range
+const range = (start, end) => Array.from({ length: end - start + 1 }, (_, index) => index + start);
+
+const generateDistrictReportConfig = ({ educationLevel }) => {
+  return {
+    fetch: {
+      dataElements: [
+        `gir_district_${educationLevel}_m`,
+        `gir_district_${educationLevel}_f`,
+        `gir_district_${educationLevel}_t`,
+      ],
+    },
+    transform: [
+      {
+        transform: 'select',
+        "'name'": `translate($row.dataElement, { gir_district_${educationLevel}_m: 'Male', gir_district_${educationLevel}_f: 'Female', gir_district_${educationLevel}_t: 'Total' })`,
+        '...': ['value', 'dataElement'],
+      },
+      'keyValueByDataElementName',
+      {
+        transform: 'aggregate',
+        name: 'group',
+        '...': 'sum',
+      },
+      {
+        transform: 'select',
+        "'Gross Intake Ratio'": `sum([$row.gir_district_${educationLevel}_m, $row.gir_district_${educationLevel}_f, $row.gir_district_${educationLevel}_t])`,
+        '...': ['name'],
+      },
+    ],
+  };
+};
+
+const generateProvinceReportConfig = ({ educationLevel }) => {
+  return {
+    fetch: {
+      dataElements: [
+        `gir_district_${educationLevel}_m`,
+        `gir_district_${educationLevel}_f`,
+        `gir_district_${educationLevel}_t`,
+      ],
+      aggregations: [
+        {
+          type: 'MOST_RECENT',
+          config: {
+            dataSourceEntityType: 'sub_district',
+          },
+        },
+      ],
+    },
+    transform: [
+      'keyValueByDataElementName',
+      {
+        transform: 'select',
+        "'Male'": `$row.gir_district_${educationLevel}_m`,
+        "'Female'": `$row.gir_district_${educationLevel}_f`,
+        "'Total'": `$row.gir_district_${educationLevel}_t`,
+        "'name'": '$row.organisationUnit',
+      },
+      {
+        transform: 'aggregate',
+        name: 'group',
+        '...': 'sum',
+      },
+    ],
```

Comment:
Please add a sort here so that Total is last (previous comment was in the wrong spot)